hw9/phone_book_json.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+try:
```

Comment:
You don't need to open the file twice here. It is enough to try to open in once and if the file does not exist you can initialize your `my_phone_book` dict as an empty dictionary.
Also, to read and transform your data to a dict you need:
- read a data from file: `data = file.read()`
- load data from json format using json lib: `my_phone_book` = json.loads(data)`

as the result, you will have something like this:
```
try:
    with open('json_pb.json', 'r+') as file_open:
        data = file.read()
        my_phone_book = json.loads(data)
except FileExistsError:
    my_phone_book = {}